### ISSUE TITLE
Issue with puppeteer

### ISSUE THREAD

**Issue by serut** [meteortesting/meteor-browser-tests#20]
I've a new issue that occurs sometimes on CircleCI : 
https://circleci.com/gh/serut/meteor-coverage-app-exemple/135

```
<server tests>
Meteor-coverage is saving client side coverage to the server. Client js files saved  {"SUCCESS":5,"FAILED":0,"TOTAL":5}
<end of client tests; then it just crash>
(node:366) UnhandledPromiseRejectionWarning: Error: Protocol error (Runtime.callFunctionOn): Target closed.
    at Promise (/home/circleci/meteor-coverage/bare-exemple/node_modules/puppeteer/lib/Connection.js:183:56)
    at new Promise (<anonymous>)
    at CDPSession.send (/home/circleci/meteor-coverage/bare-exemple/node_modules/puppeteer/lib/Connection.js:182:12)
    at ExecutionContext.evaluateHandle (/home/circleci/meteor-coverage/bare-exemple/node_modules/puppeteer/lib/ExecutionContext.js:106:44)
    at ExecutionContext.<anonymous> (/home/circleci/meteor-coverage/bare-exemple/node_modules/puppeteer/lib/helper.js:109:23)
    at WaitTask.rerun (/home/circleci/meteor-coverage/bare-exemple/node_modules/puppeteer/lib/DOMWorld.js:575:65)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
 => awaited here:
    at Function.Promise.await (/home/circleci/.meteor/packages/promise/.0.11.2.1gwulfb.vjny++os+web.browser+web.browser.legacy+web.cordova/npm/node_modules/meteor-promise/promise_server.js:56:12)
    at Promise.asyncApply (packages/meteortesting:browser-tests/browser/puppeteer.js:55:5)
    at /home/circleci/.meteor/packages/promise/.0.11.2.1gwulfb.vjny++os+web.browser+web.browser.legacy+web.cordova/npm/node_modules/meteor-promise/fiber_pool.js:43:40
(node:366) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
(node:366) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

I use the last verson of puppeteer, but on theirs issues it looks like there is a workaround.. If someone wants to dig this issue.

**Comment by SimonSimCity** [comment]
In case this helps, I tried to solve this one by this commit: https://github.com/meteortesting/meteor-browser-tests/commit/5fbcff0cf5c39f9da09e903f4e4ce4ef1cba03cc

@serut can you please write out the version you're using? I've seen that meteor doesn't update some packages if you simply call `meteor update`.

**Comment by serut** [comment]
Thanks for digging into this issue ! ;)

I'm not exactly sure about which version you're speaking about, so :   
[meteor-coverage-app-example/bare-exemple/.meteor/packages](https://github.com/serut/meteor-coverage-app-exemple/blob/master/bare-exemple/.meteor/packages)  
[meteor-coverage-app-example/bare-exemple/.meteor/versions](https://github.com/serut/meteor-coverage-app-exemple/blob/master/bare-exemple/.meteor/versions)   
and I use the last version of puppeteer

**Comment by SimonSimCity** [comment]
I've just put some effort in rewriting the way `console` events are collected from the browser. I was able to reproduce this issue and can verify that it's solved in version `1.3.0` of this plugin.

Please double-check and let me know if you face any issues.

**Comment by serut** [comment]
Yeeahh it works well ! Thanks @SimonSimCity
